src/go-build-wrapper

```diff
@@ -27,5 +27,5 @@ if ! cd "$1"; then
     exit 1
 fi
 
-go build -o "$2"
+go build -ldflags "-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" -o "$2/toolbox"
```

Comment:
Nitpick: is the `$2/toolbox` really necessary? Isn't `$2` enough?

Reply:
Yes, it is necessary. The installation `meson install` would fail to find the binary because it was created as `src/src`.

Reply:
Interesting. I see what you are saying. It does fail when built in Koji, but if I build it locally using `rpmbuild -ba toolbox.spec` then it works because it creates both `src/src` and `src/toolbox`.

What kind of dark voodoo is this?! :)